docs/how-to/monitor-your-backups.md

```diff
@@ -565,7 +565,7 @@ Resend Notification every X times = 1
 
 ## Zabbix hook
 
-<span class="minilink minilink-addedin">New in version 1.8.15</span>
+<span class="minilink minilink-addedin">New in version 1.9</span>
```

Comment:
Minor nit: It'll likely be 1.9.0. There won't be a 1.9 without a .0 or .1 or whatever.

Reply:
fixed: https://github.com/borgmatic-collective/borgmatic/pull/85/commits/cda83310c87e894703c20baddb35324dff22c8f5